### PR TITLE
Editorial: Make all spec enums lower kebab-case

### PR DESCRIPTION
**Pull request by syg**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -40040,7 +40040,7 @@ <h1>TypedArray Objects</h1>
             <dfn>%BigInt64Array%</dfn>
           </td>
           <td>
-            ~BigInt64~
+            ~bigint64~
```
Comment (syg) [inline spec.html:40043]: For `~BigInt~` and `~BigInt64~` I opted for `~bigint~` and `~bigint64~` instead of, say, `~big-int~`.

**Review by michaelficarra — APPROVED** [review]
I'm not even gonna bother doing a check for comprehensiveness, since ecmarkup will do that for us shortly. So rubber-stamping.

**Review by ljharb — COMMENTED** [review]
what about `Uint32` etc? i think all the typed array spec enum values match the casing of the constructor prior to this PR.

**Comment by syg** [comment]
> what about Uint32 etc? i think all the typed array spec enum values match the casing of the constructor prior to this PR.

IIRC we decided to display those in small caps as well.

**Comment by ljharb** [comment]
The display is fine; but if `Uint32` is fine, why not `BigInt64`?

**Comment by syg** [comment]
> The display is fine; but if Uint32 is fine, why not BigInt64?

Ah I think the missing context is along with the font change to IBM Plex, spec enums are going to be displayed as small-caps, which would've removed any capitalization in rendering anyway, thus the move the kebab-case.

**Comment by ljharb** [comment]
I’m asking why lowercase BigInt64 to bigint64 but not Uint32 to uint32?

**Comment by syg** [comment]
> I’m asking why lowercase BigInt64 to bigint64 but not Uint32 to uint32?

Oh! That would be my mistake, because I missed those.

**Comment by ljharb** [comment]
```sh
$ egrep '\~\S*\~' -o spec.html | sort | uniq
~accessor~
~all-but-default~
~all~
~ambiguous~
~ascending~
~assignment~
~async-generator~
~async-iterate~
~async~
~awaiting-return~
~backward~
~base~
~bigint64~
~bigint~
~biguint64~
~blocking~
~break~
~completed~
~constructor-method~
~continue~
~derived~
~descending~
~empty~
~end~
~enumerate~
~evaluated~
~evaluating-async~
~evaluating~
~executing~
~failure~
~field~
~float32~
~float64~
~forward~
~frozen~
~fulfilled~
~fulfill~
~generator~
~global~
~initialized~
~init~
~int16~
~int32~
~int8~
~invalid~
~iterate~
~key+value~
~key~
~lexical-binding~
~lexical-this~
~lexical~
~linked~
~linking~
~method~
~namespace-object~
~namespace~
~new~
~non-constructor-method~
~non-generator~
~non-lexical-this~
~normal~
~number~
~pending~
~read-through-holes~
~rejected~
~reject~
~return~
~sans-serif~
~sealed~
~seq-cst~
~simple~
~skip-holes~
~start+end~
~start~
~strict~
~string~
~suspended-start~
~suspended-yield~
~symbol~
~sync~
~throw~
~uint16~
~uint32~
~uint8clamped~
~uint8~
~uninitialized~
~unlinked~
~unordered~
~unresolvable~
~unused~
~value~
~var-binding~
~~
```

LGTM now, thanks

**Pull request merged**